test/e2e/service.go

```diff
@@ -245,9 +246,62 @@ var _ = Describe("Services", func() {
 		defer func() {
 			close(done)
 		}()
-	}, 120.0)
+	}, 240.0)
+	It("should correctly serve identically named services in different namespaces on different external IP addresses", func(done Done) {
```

Comment:
@zmerlynn or @satnam6502 for better context - I have not yet used this new test stuff..